pkg/security/tests/process_test.go

```diff
@@ -523,6 +524,69 @@ func TestProcessContext(t *testing.T) {
 	})
 }
 
+func TestProcessExecCTime(t *testing.T) {
+	executable := "/usr/bin/touch"
+	if resolved, err := os.Readlink(executable); err == nil {
+		executable = resolved
+	} else {
+		if os.IsNotExist(err) {
+			executable = "/bin/touch"
+		}
+	}
+
+	copy := func (src string, dst string) error {
+		in, err := os.Open(src)
+		if err != nil {
+			return err
+		}
+		defer in.Close()
+	
+		out, err := os.Create(dst)
+		if err != nil {
+			return err
+		}
+		defer out.Close()
+	
+		if _, err = io.Copy(out, in); err != nil {
+			return err
+		}
+		if err := os.Chmod(dst, 0o755); err != nil {
+			return err
+		}
+
+		return nil
+	}
+
+	ruleDef := &rules.RuleDefinition{
+		ID:         "test_exec_ctime",
+		Expression: "exec.file.change_time < 5s",
+	}
+
+	test, err := newTestModule(t, nil, []*rules.RuleDefinition{ruleDef}, testOpts{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer test.Close()
+
+	err = test.GetSignal(t, func() error {
+		testFile, _, err := test.Path("touch")
+		if err != nil {
+			t.Fatal(err)
+		}
+		copy(executable, testFile)
+
+		cmd := exec.Command(testFile, "/tmp/test")
+		return cmd.Run()
+	}, func(event *sprobe.Event, rule *rules.Rule) {
+		assert.Equal(t, rule.ID, "test_exec_ctime", "wrong rule triggered")
```

Comment:
According to https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/assert#Equal the order is expected then actual. I'm working on a PR to change this globally (#8643)